changes private view (#814)
akeneo/pim-helpcenter: content/md/products-super-power/manage-your-views.md

To create your first view:
1.  Click on the "..." nearby the view selector menu
![Create a view](../img/Products_CreateView.png)

> 1.  Select `Create view` in the dropdown. A window will appear.

Reviewer: ```suggestion
1.  Select `Create view` in the dropdown menu. A window will appear.
```

1.  Enter a name for your view

> 1.  Select `public view` if you want to share this view with all other users. If it's not selected, the view is private.

Reviewer: ```suggestion
1.  Select `public view` if you want to share this view with every user. If it's not selected, the view is private.
```
